templates/netlify/README.md

```diff
@@ -1,7 +1,7 @@
 # Welcome to Remix!
```

Comment:
This template should be a repo somewhere on the Netlify GH org. This `templates` dir is a temporary spot for legacy v1 templates. No new templates should be created here.